src/index.html

```diff
@@ -3,12 +3,64 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a href="#home"
+        class="header__icon">
+        <img src="./images/logo.png"
+        alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Laptops & Computers"
+              class="nav__link"
+              data-qa="hover">
+              Laptops & Computers
+            </a>
```

Comment:
```suggestion
            <a 
              href="#Laptops & Computers"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```
Fix all cases
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/8e5b7335-e7cf-4e50-9f54-d3a8c37d5286)